src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.io.*;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String information;
+        int buy = 0;
+        int supply = 0;
+
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName));
```

Comment:
use try with resources with reader, writer should be in a separate method dedicated solely to writing

Reply:
not fixed - use try with resouces